### ISSUE TITLE
Outdated NoUiSlider Script - Conflict with other plugins

### ISSUE THREAD

**Issue by Tekanewa** [reduxframework/redux-framework#1631]
Hi,

the Redux Framework includes the pretty awesome NoUiSlider script to create slider inputs. Unfortunately, it still uses the VERY outdated v5.0 of the script, while the current official release is 7.0. 

http://refreshless.com/nouislider/

The changes between the versions are in some part significant as the script now requires some different parameters to be initialized, making the old version (5.0) incompatible with the new one (7.0).

When using another plugin that embeds itselves into the WP backend process and if that other plugin is already using the current version of NoUiSlider, while using a theme or other plugin that is using Redux, a JS error is produced, which prevents at least one of the two script versions from working, resulting in broken inputs and/or other issues.

Long story short, is it possible (and probably even time) to update the NoUiSlider version utilized by Redux to its current release in order to remain compatible?

Thanks!

Kalydon
